src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -7954,11 +7954,34 @@ public void CS0501ERR_ConcreteMissingBody02()
     protected abstract object S { set; } // no error
 }
 ";
-            CreateCompilation(text).VerifyDiagnostics(
+            // IMPORTANT: There used to be CS0501 (missing body) error on P and Q properties.
+            // This has changed with semi auto properties, and it was decided (for all lang versions) to keep
+            // the new logic for determining if a property is auto property or not.
+            // See https://github.com/dotnet/roslyn/pull/57076#discussion_r753751334
+
+            // PROTOTYPE(semi-auto-props): Unassigned field warning?
+
+            CreateCompilation(text, parseOptions: TestOptions.Regular10).VerifyDiagnostics(
                 // (3,20): error CS0501: 'C.P.get' must declare a body because it is not marked abstract, extern, or partial
-                Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "get").WithArguments("C.P.get"),
+                //Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "get").WithArguments("C.P.get"),
                 // (4,38): error CS0501: 'C.Q.set' must declare a body because it is not marked abstract, extern, or partial
-                Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "set").WithArguments("C.Q.set"),
+                //Diagnostic(ErrorCode.ERR_ConcreteMissingBody, "set").WithArguments("C.Q.set"),
+                // (5,30): warning CS0626: Method, operator, or accessor 'C.R.get' is marked external and has no attributes on it. Consider adding a DllImport attribute to specify the external implementation.
+                Diagnostic(ErrorCode.WRN_ExternMethodNoImplementation, "get").WithArguments("C.R.get"));
+        }
+
+        [Fact]
+        public void CS0501ERR_ConcreteMissingBody02_SemiAutoProps()
+        {
+            var text = @"abstract class C
+{
+    public int P { get; set { } }
+    public int Q { get { return 0; } set; }
+    public extern object R { get; } // no error
+    protected abstract object S { set; } // no error
+}
+";
+            CreateCompilation(text, parseOptions: TestOptions.RegularPreview).VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=78&length=17)

I am assuming we expect some errors eventually reported for this new scenario. If so, a PROTOTYPE comment should be added.

Reply:
@AlekseyTs This is actually not a new scenario. I copied the test and forgot to modify it.

What errors are you expecting for this scenario?

Reply:
>Going to remove it altogether.

>What errors were you expecting for this scenario?
 
It wasn't clear to me why the test was added. I though that you expect some behavior change for this scenario. Now or in the future. Removing is fine if that is not the case.